documentation/projects/proposals/trust_and_safety/content_report_moderation/20230517-implementation_plan_mature_copy_change.md

```diff
@@ -70,8 +81,40 @@ value) in `AbstractMediaReport` definition will also need to be changed to
 `sensitive` and `sensitive_filtered` respectively. The migration necessary for
 changing the data is described below, however the API will need to continue to
 accept `mature` as a report reason and convert it internally to `sensitive` (at
-least until we decide to make a version change to the API). The media report
-tables already use table aliases as well
+least until we decide to make a version change to the API).
+
+In order to perform both of these changes in a
+[zero-downtime manner](https://docs.openverse.org/general/zero_downtime_database_management.html),
+the following steps will need to be taken:
+
+1. Rename the models (without changing the underlying tables) and associated
+   references for:
+   - `AbstractMatureMedia` -> `AbstractSensitiveMedia`
+   - `MatureImage` -> `SensitiveImage`
+   - `MatureAudio` -> `SensitiveAudio`
+2. Add `sensitive` and `sensitive_filtered` as a new status and reason
```

Comment:
We can actually skip the changes to report status as well, we just need to update `reason`. #3494 removes the report status altogether in favour of a new moderation decision table that records the action taken from one or many reports.